src/Passes/docs/src/UserGuide/ApplyingAProfile.md

```diff
@@ -0,0 +1 @@
+# Applying a profile
```

Comment:
Is there a TODO or GH Issue for tracking this?

Reply:
TODO was added in the TOC and this document was removed for the time being.